padertorch/modules/recurrent.py

```diff
@@ -12,7 +12,8 @@ def __init__(
             num_layers: int = 1,
             bidirectional: bool = False,
             dropout: float = 0.,
-            batch_first: bool = True
+            batch_first: bool = True,
+            save_states: bool = False
```

Comment:
Should the default for the class `StatefulLSTM` be, that it is statefull?

Reply:
you are right, I'll switch that and remove the defaults for input_size and hidden_size since they are quiet random